app/models/project.rb

```diff
@@ -2,20 +2,31 @@
 #
 # Table name: projects
 #
-#  id          :bigint           not null, primary key
-#  description :string
-#  lang        :enum             default("unassigned")
-#  name        :string
-#  created_at  :datetime         not null
-#  updated_at  :datetime         not null
-#  github_id   :integer          not null
+#  id            :bigint           not null, primary key
+#  description   :string
+#  lang          :enum             default("unassigned")
+#  name          :string
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#  department_id :bigint
+#  github_id     :integer          not null
+#
+# Indexes
+#
+#  index_projects_on_department_id  (department_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (department_id => departments.id)
 #
 
 class Project < ApplicationRecord
   enum lang: { ruby: 'ruby', python: 'python', nodejs: 'nodejs',
-               react: 'react', ios: 'ios', android: 'android',
+               ios: 'ios', android: 'android', reactnative: 'react_native',
```

Comment:
```suggestion
               ios: 'ios', android: 'android', react_native: 'react_native',
```

im not sure if it is a mistake or you did it for a reason

Reply:
I preffer to keep it with underscore, so in the future we can just do a humanize or something like that, and obtain ReactNative. Maybe we should do the same with nodejs and vuejs